src/h5web/toolbar/controls/DomainSlider/DataHistogram.tsx

```diff
@@ -52,22 +57,33 @@ function DataHistogram(props: Props) {
           ))}
         </svg>
 
-        <AxisBottom
-          top={height}
-          scale={xScale}
-          numTicks={adaptedNumTicks(width)}
-          tickClassName={styles.tick}
-          tickComponent={Tick}
-          tickFormat={xScale.tickFormat(adaptedNumTicks(width), '.3~g')}
-        />
         <AxisLeft
           scale={yScale.range([height, 0])}
           numTicks={adaptedNumTicks(height)}
           tickClassName={styles.tick}
           tickComponent={Tick}
           tickFormat={yScale.tickFormat(adaptedNumTicks(height), '.3~g')}
         />
+        {!colorMap && (
+          <AxisBottom
+            top={height}
+            scale={xScale}
+            numTicks={adaptedNumTicks(width)}
+            tickClassName={styles.tick}
+            tickComponent={Tick}
+            tickFormat={xScale.tickFormat(adaptedNumTicks(width), '.3~g')}
+          />
+        )}
```

Comment:
If `colorMap` is defined, the Axis is part of `ColorBar`. Else, we display a `AxisBottom`.